platform_versioned_docs/version-24.1.0/reports/overview.mdx

```diff
@@ -30,6 +30,12 @@ Create a config file that defines the paths to a selection of output files publi
 Any configuration provided in the interface will override configuration supplied in the pipeline repository.
 :::
 
+### Configure reports for Nextflow CLI runs 
+
+Seqera can manage the reports and log files for pipeline runs launched with Nextflow CLI (`nextflow run <pipeline> -with-tower`) when the run is launched with a workspace primary compute environment. Specify the Seqera workspace from which to run the pipeline by setting the `TOWER_WORKSPACE_ID` environment variable. This workspace must have a primary compute environment set. Reports are listed under the **Reports** tab on the run details page. 
+
```

Comment:
The run is not launched with a CE from platform . The run is launched with `nextflow run ... -with-tower` then the logs and reports will be visualized if the primary CE has access to those flies.

Reply:
Right, but it's executed with a CE that is also a primary CE in SP. I've adjusted the language a bit. Let me know if this is clearer.